misc/graylog.conf

```
@@ -518,6 +518,9 @@ inputbuffer_wait_strategy = blocking
 # Enable the message journal.
 message_journal_enabled = true
 
+# since we allow space and slash as valid characters in field names, the cleanKey function in pipelines needs to have a switch to be backwards compatible for users that depend on it
+clean_key_keeps_space_and_slash = true
```

Nit: Maybe prefix it with `pipeline_` or even `pipeline_rules_`?

```suggestion
pipeline_clean_key_keeps_space_and_slash = true
```